src/components/chat/FadeIn.tsx

```
@@ -0,0 +1,18 @@
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    // Fade in after the component mounts
+    setIsVisible(true);
+  }, []);
+
+  return (
+    <div
+      className={`transition-opacity duration-${duration} ${isVisible ? 'opacity-100' : 'opacity-0'}`}
+    >
+      <p dangerouslySetInnerHTML={{ __html: text }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)